save_test.go

```diff
@@ -0,0 +1,96 @@
+package zstash
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFilterExistingPaths(t *testing.T) {
```

Comment:
This is an implementation detail. I wonder if we should start by adding an test case for the Save func instead of this internal function?